### DISCUSSION TITLE
ISSUE: Images are not rendering in IOS 16.5.1

### DISCUSSION THREAD

**Discussion by charan-web** [vercel/next.js#52114] [Help]
### Summary

I am writing to bring to your attention an urgent issue we have encountered with image rendering on our recently deployed website in production.

After a thorough evaluation, we have observed that the images on our website are not rendering correctly on devices running iOS 16.5.1, the latest version of the operating system. However, we have confirmed that on older versions of iOS (prior to 16.5), the images display properly without any issues.

We have taken the following steps to investigate the matter:

* Verified image file formats: We have ensured that the images used on our website are in widely supported formats such as JPEG, PNG, or GIF.

* Checked image paths and URLs: We have double-checked the accuracy and accessibility of the image paths and URLs used in our website's code, ensuring there are no typographical errors and that the images are located in the correct directories.

* Examined caching and compression settings: We have reviewed the caching and compression settings on our server and within our website's code to ensure they are correctly configured and not causing any conflicts with image rendering.

* Tested on different iOS devices and browsers: We have conducted tests by accessing our website on various iOS devices running different versions of the operating system, as well as using different web browsers. This was done to determine whether the issue is specific to iOS 16.5.1 or if it affects other combinations as well.

* Debugged using developer tools: We have employed browser developer tools and remote debugging features on iOS devices to inspect the network requests and responses related to image loading. We are actively looking for any error messages or warnings that could shed light on the problem.

 Given the critical nature of this issue, we kindly request your expertise and assistance in resolving this matter as quickly as possible. If you could provide any guidance, suggestions, or potential solutions, we would greatly appreciate it.

Please let us know if you require any additional information or if there is anything else we can provide to assist in the troubleshooting process. We are committed to working closely with you to ensure the timely resolution of this problem
``` 
import Image from "next/image";
 <Image
                                    src={getImageUrl(reports[0]?.thumbnail)}
                                    alt={reports[0]?.webTitle}
                                    width={660}
                                    height={400}
                                    priority={true}
                               />
``` 

Thank you for your attention and support. We eagerly await your prompt response.

### Additional information

```js
<Image
                                    src={getImageUrl(reports[0]?.thumbnail)}
                                    alt={reports[0]?.webTitle}
                                    width={660}
                                    height={400}
                                    priority={true}
                                />
```

**Comment by icyJoseph** [comment]
> Debugged using developer tools: We have employed browser developer tools and remote debugging features on iOS devices to inspect the network requests and responses related to image loading. We are actively looking for any error messages or warnings that could shed light on the problem.

Got a link to the project? Did the images just plain not load when you used dev tools?
